datasets/opus_100/README.md

```diff
@@ -0,0 +1,251 @@
+---
+task_categories:
+- sequence-modeling
+multilinguality:
+- translation
+task_ids:
+- language-modeling
+languages:
```

Comment:
The languages depend on the configuration that is used. Therefore can you group the languages by config name here ?

For example:
```
languages:
  en-es:
  - en
  - es
  en-hi:
  - en
  - hi
```

Could you change this please ?